src/CSnakes.Tests/GeneratedSignatureTests.cs

```diff
@@ -50,6 +50,7 @@ public class GeneratedSignatureTests
     [InlineData("def hello(a: Union[int, str] = 5) -> Any:\n ...\n", "PyObject Hello(PyObject? a = null)")]
     [InlineData("def hello(data: Literal[1, 'two', 3.0]) -> None:\n ...\n", "void Hello(PyObject data)")]
     [InlineData("def hello(n: None = None) -> None:\n ...\n", "void Hello(PyObject? n = null)")]
+    [InlineData("def hello(n: Foo = ...) -> None:\n ...\n", "void Hello(PyObject? n = null)")]
```

Comment:
`n: Foo = ...` should translate to `PyObject n`. I can understand if `n` was type-annotated as `Foo | None`, but `Foo` says that `None` isn't expected.